bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/twinguard/BoschTwinguardHandler.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.boschshc.internal.devices.twinguard;
+
+import static org.openhab.binding.boschshc.internal.devices.BoschSHCBindingConstants.*;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.boschshc.internal.devices.BoschSHCHandler;
+import org.openhab.binding.boschshc.internal.devices.twinguard.dto.AirQualityLevelState;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.RefreshType;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonSyntaxException;
+
+/**
+ * The {@link BoschSHCHandler} is responsible for handling commands for the TwinGuard handler.
+ *
+ * @author Stefan Kästle - Initial contribution
+ */
+@NonNullByDefault
+public class BoschTwinguardHandler extends BoschSHCHandler {
+
+    public BoschTwinguardHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        Bridge bridge = this.getBridge();
+
+        if (bridge != null) {
+            logger.debug("Handle command for: {} - {}", channelUID.getThingUID(), command);
+
+            if (command instanceof RefreshType && CHANNEL_TEMPERATURE.equals(channelUID.getId())) {
+                // Only refresh the state for CHANNEL_TEMPERATURE, the rest will be filled in too.
+                AirQualityLevelState state = this.getState("AirQualityLevel", AirQualityLevelState.class);
+                if (state != null) {
+                    updateAirQualityState(state);
+                }
+            }
```

Comment:
Are you going to handle refresh of any of the other channels?

Reply:
Removed the channel check, in my opinion the command should be sent no matter which channel should be refreshed.